ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -189,3 +189,30 @@ def test_numpy_rfftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.rfft2",
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
+        min_num_dims=0 or None,
+        max_num_dims=2,
+        force_tuple_axis=True
+    ),
+    norm=st.sampled_from(["backward", "ortho", "forward"]),
+    s=st.lists(st.integers()),
```

Comment:
We also need to test for cases where `s` is None. You can use `st.one_of` to combine more than 1 strategies and pick one of them per each test run